ci-operator/step-registry/cucushift/installer/check/compact/cucushift-installer-check-compact-commands.sh

```diff
@@ -20,6 +20,11 @@ stderr=$(mktemp)
 stdout=$(mktemp)
 oc get nodes --no-headers 1>${stdout} 2>${stderr} || true
 
+if [[ -n ${REMOTEWORKER_NUMBER} ]]; then
```

Comment:
Actually I think this step has nothing with remote worker installation, if the step should be run against remote worker install workflow, I suggest to not call the `cucushift-installer-check` chain, but reference the needed ref steps in the remote worker install workflow.